Sami.s.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "sami.s"
+    },
+    "record": {
+        "AAAA": "https://github.com/Sami9889
```

Comment:
File does not follow proper schema. See docs for more information: https://docs.is-a.dev/